ports/tensorflow-cc/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "tensorflow-cc",
-  "version-string": "2.3.1",
-  "port-version": 2,
+  "version-string": "2.4.1",
```

```suggestion
  "version-semver": "2.4.1",
```
The package version should be `version-smver`, please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions.